mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -108,6 +110,14 @@ class RegressionMetrics @Since("2.0.0") (
     math.sqrt(this.meanSquaredError)
   }
 
+  @Since("2.0.0")
```

Comment:
This is in the wrong place, and use `@return` below.
The only problem with this PR is that it introduces more computation per example that it processes, even when it's not used.